ddtrace/tracer.py

```diff
@@ -157,6 +157,15 @@ def start_span(self, name, child_of=None, service=None, resource=None, span_type
                 resource=resource,
                 span_type=span_type,
             )
+
+            #    http://pypi.datadoghq.com/trace/docs/#distributed-tracing
+            parent_trace_id, parent_span_id = context._get_parent_span_ids()
```

Comment:
it's kinda like you want to move the parent trace_id/span_id logic to a separate function.